tcwebhooks-web-ui/src/main/resources/buildServerResources/WebHook/js/editWebhook.js

```diff
@@ -539,19 +539,23 @@ function populateWebHookDialog(id){
 			$j.each(webhook.builds, function(){
 				var thing = $j(this.buildTypeName).text();
```

Comment:
This was the line that was causing the issue. Removing it fixes the bug you're seeing.

Reply:
Sigh.. I saw that line and completely ignored it because the code below I wanted to change first, and forgot to look at it later. Yeah, that would cause some invalid expressions. Any reason it isn't just `console.log(this.buildTypeName);` instead?

Reply:
I imagine it was part of the original development which was never removed.